source/include/mqtt_agent.h

```diff
@@ -398,8 +685,14 @@ MQTTStatus_t MQTTAgent_Ping( const MQTTAgentContext_t * pMqttAgentContext,
  * @brief Add a command to call MQTT_Connect() for an MQTT connection. If a session
  * is resumed with the broker, it will also resend the necessary QoS1/2 publishes.
  *
+ * @note The MQTTAgent_Connect function is provided to give a thread safe equivalent
+ * to the MQTT_Connect API. However, it is unlikely to be used as the MQTT connection
+ * is likely to be created by the agent task before calling MQTTAgent_CommandLoop().
```

Comment:
Can you add some more language about how it is recommended that the agent task create the connection instead of other tasks?

Reply:
Updated the text to match tone of the `MQTT_Disconnect` API doc.